service/storage_service.cc

```diff
@@ -5666,8 +5666,10 @@ future<raft_topology_cmd_result> storage_service::raft_topology_cmd_handler(raft
                 break;
             }
         }
+    } catch (const raft::request_aborted& e) {
+        rtlogger.warn("raft_topology_cmd {} failed with: {}", cmd.cmd, e);
```

Comment:
We usually print something like "raft_topology_cmd {} aborted" in such cases. I think it's a bit better (short and understandable), but I don't think I would resend this PR just for this change.

Reply:
I won't resend then.